docs/controls/access_control/ac-05.md

```diff
@@ -0,0 +1,53 @@
+# Access Control
+## AC-05 - Separation of Duties
+
+The organization:  
+a. Separates [at a minimum, data creation and control, software development and maintenance, and security functions];
+
+b. Documents separation of duties of individuals; and 
+
+c. Defines information system access authorizations to support separation of duties.
+
+### TDP Implementation
+
+**Part a.**  
+The TANF Data Portal (TDP) system has explicitly defined the roles and responsibilities that accounts perform to operate and maintain the system. This includes data creation and processing, software development, maintenance, and security implementation.
+
+Every role has flags associated with the account.  These flags are *SuperUser* and *Staff* and are set to *False* by default.  When a new TDP account is created, the OFA Admin has the ability to set the flags to *True* based on the user's job responsibilities.  Note: if an account has the *Staff* flag set to *True*, the account will have access to Django Admin, but will not have access to any of the account management functionality.
+
+The TDP system has two roles: OFA Admin and Data Prepper.
+  * OFA Admin has the following abilities: security functionality to manage TDP accounts through Django Admin, upload data on behalf of Data Preppers, upload data files locally into the web application
+  * OFA Admin *SuperUser* and *Staff* flags are both set to *True*.  This gives the OFA Admin the ability to access Django Admin and access to account management functionality.
```

Comment:
This seems to conflict with the sentence [below](https://github.com/HHS/TANF-app/pull/174/files#diff-9745bc4113bf26ff62eb6a3bd1d2f33ae9541df4cf562274fe1ab5a95ea13cebR28):
> Other users with the role OFA Admin would not have Staff and SuperUser which separates the System Owner from OFA Admin.

Reply:
Yes, this is not true. OFA Admin *SuperUser* and *Staff* flags are set to false by default.

Reply:
Updated to state that the OFA Admin SuperUser and Staff flags set to false by default.